compiler-rt/test/asan/TestCases/Posix/deep_call_stack.cpp

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: shell
```

Comment:
nit: i would prefer if the comment stayed in the first line

Reply:
Done.